### ISSUE TITLE
Implement Settlers tile improvements

### ISSUE THREAD

**Issue by SWY1985** [SWY1985/CivOne#43]

**Comment by SWY1985** [comment]
Reported by DracoNoir:
Unable to build fortresses with settlers even after discovering construction (WIP?)

**Comment by kazgaroth011** [comment]
I wanted to open a new issue but I see DracoNoir already mentioned this. Building fortresses should be implemented (only after Construction is discovered).

**Comment by SWY1985** [comment]
I guess I reintroduced an old bug then.
